src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -552,7 +561,8 @@
 
 {% block switch_widget %}
   {% apply spaceless %}
-  <div class="input-group">
+  {% set rowAttributes  = row_attr|default({})%}
+  <div class="input-group {{ rowAttributes.class|default('') }}" {% for key, rowAttr in rowAttributes %} {% if key != 'class' %}{{ key }}="{{ rowAttr }}"{% endif %}{% endfor %}>
```

Comment:
This looks a lot like the `widget_attributes` so I'm thinking maybe we could create a similar `row_widget_attributes` that could be used in other widgets as well